src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString.split(';').map((pair) => {
```

Comment:
Avoid using variable names like 'result' as it decreases code readability. Consider using a more descriptive name.